src/Symfony/Bridge/Twig/Tests/Extension/AbstractBootstrap3LayoutTestCase.php

```diff
@@ -2141,7 +2141,7 @@ public function testMoney()
     public function testMoneyWithoutCurrency()
     {
         $form = $this->factory->createNamed('name', 'Symfony\Component\Form\Extension\Core\Type\MoneyType', 1234.56, [
-            'currency' => false,
+            'currency' => null,
```

Comment:
Passing `false` is actually what is documented as the way to suppress showing the currency symbol: 

> You can also set this to false to hide the currency symbol.

see https://symfony.com/doc/current/reference/forms/types/money.html#currency

Reply:
/cc @Girgias